packages/nouns-webapp/package.json

```diff
@@ -64,9 +69,12 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "yarn i18n:compile && react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "i18n:extract": "lingui extract",
+    "i18n:compile": "lingui compile",
```

Comment:
Add `"prepare": "yarn i18n:compile",` so localization works out of the box after installing dependencies for local development.